recipes/peptidedb/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.0.1" %}
+{% set version_string = "0.0.1-beta" %}
```

Comment:
if the version of the package is 0.0.1 but the version_string is 0.0.1-beta ... what happens if you are out of beta? Will this be 0.0.2? We usually recommend to only package stable software. Is that possible here?